__DEFINES/setup.dm

```diff
@@ -297,6 +297,14 @@ var/MAX_EXPLOSION_RANGE = 14
 #define ALL ~0
 #define NONE 0
 
+//sharpness flags
+#define SHARP_TIP 		1 // Has a pointy-stabby end, such as a syringe or a knife tip.
+#define SHARP_BLADE		2 // Has a blade long and thin enough to slice something with.
+#define SERRATED_BLADE	4 // Has saw-like teeth to cut through harder materials, however messily. The serrated edge may not necessarily be sharp!
+#define CHOPWOOD		8 // Kind of an abstract one: The implement is suitable to chop wood with. Essentially a saw or something big enough.
+#define INSULATED_EDGE 	16 // One of the edges of this thing is insulated, even though the rest of it isn't.
```

Comment:
Glass spears for example

The spear itself is mostly conductive so if you chuck it at a shocked airlock it should spark and other similar behaviors

The spearhead is insulated so if you ATTACK something with it it shouldn't conduct